pkg/controllers/management/drivers/nodedriver/machine_driver.go

```diff
@@ -136,6 +169,32 @@ func (m *Lifecycle) download(obj *v3.NodeDriver) (*v3.NodeDriver, error) {
 	if err != nil && !errors.IsAlreadyExists(err) {
 		return obj, err
 	}
+	return m.createCredSchema(obj, credFields)
+}
+
+func (m *Lifecycle) createCredSchema(obj *v3.NodeDriver, credFields map[string]v3.Field) (*v3.NodeDriver, error) {
+	name := credentialConfigSchemaName(obj.Spec.DisplayName)
+	_, err := m.schemaLister.Get("", name)
+	if err != nil && !errors.IsNotFound(err) {
+		return obj, err
+	} else if errors.IsNotFound(err) {
```

Comment:
what if error is nil here? does IsNotFound handle it?

Reply:
thank you! updated